doc/Caliptra.md

```diff
@@ -863,27 +863,27 @@ A Caliptra RTM shall follow/implement the secure boot guidelines as described in
+To capture all firmware and configuration changes, Caliptra tracks and attests to both current measurements and cumulative measurement PCR banks. The current measurement is a snapshot of the currently running firmware and configuration. This provides easy reference for the current version. If the current and cumulative measurements are different, it can safely be assumed that the device has undergone some update. The cumulative measurement captures all of the firmware and state transitions from a clean cold boot to the current version. The cumulative measurement must be accompanied by a log structure that describes the path from boot to current measurement using hash extensions. A verifier can understand a device’s path to its current state by replaying log entries to reconstruct the cumulative measurement.
 
-The log and cumulative measurement mechanism is similar to that used in TPM.  In this model, Caliptra only needs to securely manage the measurements; the log does not need to be secured or maintained by Caliptra or the SoC.  The construction of measurements through cryptographic hash extensions means the log must provide the exact order and evidence needed to reconstruct the measurement.  As such, the log is tamper evident by design and does not need to be kept secure.
+The log and cumulative measurement mechanism is similar to that used in TPM. In this model, Caliptra only needs to securely manage the measurements; the log does not need to be secured or maintained by Caliptra or the SoC. The construction of measurements through cryptographic hash extensions means that the log must provide the exact order and evidence needed to reconstruct the measurement. As such, the log is tamper evident by design and does not need to be kept secure.
 
-Caliptra contains 32 PCR 384 banks that are extendable by the SHA engine, and readable by Caliptra firmware.  The usage of the PCR Banks is as follows:
+Caliptra contains 32 PCR 384 banks that are extendable by the SHA engine, and readable by Caliptra firmware.  The usage of the PCR banks is as follows:
 
-*Table 9: PCR Bank Usage*
+*Table 9: PCR bank usage*
 
-|**PCR Number**|**Type**|**Extend Ctrl**|**Description**|
+|**PCR number**|**Type**|**Extend Ctrl**|**Description**|
 | - | - | - | - |
-|PCR0|Cumulative|ROM|Holds Caliptra's FMC firmware update journey; including state and FMC, and measurements extended by ROM.|
-|PCR1|Current|ROM|Caliptra’s boot FMC, and and state. This is a snapshot of the current state.|
-|PCR2|Cumulative|FMC|Holds Caliptra's runtime firmware update journey; including state and FMC, and measurements extended by FMC.|
+|PCR0|Cumulative|ROM|Holds Caliptra's FMC firmware update journey. This includes state and FMC, and measurements extended by ROM.|
```

Comment:
These descriptions are now swapped actually. Should read:

```
|PCR0|Current|ROM|Caliptra’s boot FMC, and state. This is a snapshot of the current state.|
|PCR1|Cumulative|ROM|Holds Caliptra's FMC firmware update journey. This includes state and FMC, and measurements extended by ROM.|
|PCR2|Current|FMC|Caliptra’s boot and runtime firmware measurement and state.  This is a snapshot of the current state.|
|PCR3|Cumulative|FMC|Holds Caliptra's runtime firmware update journey. This includes state and FMC, and measurements extended by FMC.|
```

There are some other changes in comments that follow, but they might be out of scope.

Reply:
fixed in merge